src/components/ThankYou/ThankYou.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import styles from './ThankYou.sass';
+
+const ThankYou = () => {
+  return (
+    <div className={styles.thankYouWrapper}>
+      <h1> Thank you! Your submission has been submitted for review. </h1>
+      <Link to="/"> View some our event listing. </Link>
```

Comment:
Should be "View some of our event listings", or, IMO, "View our event listings".